app/views/claims/subjects_taught.html.erb

```diff
@@ -1,39 +1,24 @@
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
     <%= form_for current_claim, url: claim_path do |form| %>
-      <%= form.hidden_field :mostly_teaching_eligible_subjects %>
       <%= form_group_tag current_claim do %>
-        <fieldset class="govuk-fieldset">
+        <fieldset class="govuk-fieldset" id="tslr_claim_eligible_subjects">
           <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
             <h1 class="govuk-fieldset__heading">
-              <%= t("tslr.questions.mostly_teaching_eligible_subjects") %>
+              <%= t("tslr.questions.subjects_taught") %>
             </h1>
           </legend>
-          <p class="govuk-body">
-            Eligible subjects are:
-          </p>
-          <ul class="govuk-list govuk-list--bullet">
-            <li>biology</li>
-            <li>chemistry</li>
-            <li>physics</li>
-            <li>computer science</li>
-            <li>languages (not including English)</li>
-          </ul>
-          <p class="govuk-body">
-            <span class="govuk-hint">
-              If you've been off on long-term leave or sick, include the time you were scheduled to teach these subjects.
-            </span>
-          </p>
-          <%= errors_tag current_claim, :mostly_teaching_eligible_subjects %>
-          <div class="govuk-radios">
-            <div class="govuk-radios__item">
-              <%= form.radio_button(:mostly_teaching_eligible_subjects, true, class: "govuk-radios__input") %>
-              <%= form.label :mostly_teaching_eligible_subjects_true, "Yes", class: "govuk-label govuk-radios__label" %>
-            </div>
-            <div class="govuk-radios__item">
-              <%= form.radio_button(:mostly_teaching_eligible_subjects, false, class: "govuk-radios__input") %>
-              <%= form.label :mostly_teaching_eligible_subjects_false, "No", class: "govuk-label govuk-radios__label" %>
+          <div class="govuk-checkboxes">
+            <% TslrClaim.eligible_subjects.each do |subject, value| -%>
+              <div class="govuk-checkboxes__item">
+                <%= check_box_tag "tslr_claim[eligible_subjects][]", subject, current_claim.eligible_subjects.include?(subject), class: "govuk-checkboxes__input", id: "eligible_subjects_#{subject}"  %>
+                <%= label_tag "tslr_claim[eligible_subjects][]", t("tslr.questions.eligible_subjects.#{subject}"), class: "govuk-label govuk-checkboxes__label", for: "eligible_subjects_#{subject}"  %>
+              </div>
+            <% end %>
+            <div class="govuk-checkboxes__item">
+              <%= form.check_box(:mostly_teaching_eligible_subjects, {class: "govuk-checkboxes__input"}, false, '')  %>
+              <%= form.label :mostly_teaching_eligible_subjects_false, "Not applicable, I didn't teach any of these subjects", class: "govuk-label govuk-checkboxes__label" %>
```

Comment:
Is "Not applicable" actually an answer to this question? Isn't it "No, I didn't teach any of these subjects", or just "I didn't teach any of these subjects"?